lib/facter/util/partitions.rb

```diff
@@ -44,4 +44,16 @@ def self.label(partition)
   def self.available?
     !self.list.empty?
   end
+
+  def self.flushable?
```

Comment:
Rather than having two different methods each implementation needs to implement, I think I'd prefer to see each implementation simply implement a `flush!` method and any implementations that don't need flushing simply do nothing.  It's less churn and harder for some theoretical new implementation (ignoring the fact 2.x is in maintenance mode) to accidentally implement one without the other.  Ideally a "base" implementation would implement the no-op flush, but it isn't worth refactoring this to make that happen.